### PR TITLE
Create restaurant

### PR DESCRIPTION
**Pull request by nguyentrison2585**
==> Title: Creating restaurant

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/19361?issue_count=21&issue_position=14&next_issue_id=19359&prev_issue_id=19362

==> Attachment File:
image

### CODE REVIEW

**Comment by nguyentrison2585** [comment]
Ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -8,4 +8,8 @@ def page_index params_page, index, per_page
     params_page ||= 1
     (params_page.to_i - 1) * per_page + index + 1
   end
+
+  def correct_restaurant? restaurant
+    return true if @restaurant &.id == restaurant.id
```
Comment (phucmh-1356) [inline app/helpers/application_helper.rb]:
```suggestion
    return true if @restaurant&.id == restaurant.id
```
sao k dùng cancancan đi em

File: app/helpers/restaurants_helper.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  def show_navigation restaurant
+    "show" if correct_restaurant?(restaurant)
+  end
+
+  def active_navigation restaurant
+    "active" if correct_restaurant?(restaurant)
+  end
+
+  def active_update_restaurant restaurant
+    "active" if correct_restaurant?(restaurant) && params[:action] == "edit"
```
Comment (phucmh-1356) [inline app/helpers/restaurants_helper.rb]:
```suggestion
    "active" if correct_restaurant?(restaurant) && ["edit", "orders"].include? params[:action]
```

**Comment by nguyentrison2585** [comment]
Ready

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -8,4 +8,8 @@ def page_index params_page, index, per_page
     params_page ||= 1
     (params_page.to_i - 1) * per_page + index + 1
   end
+
+  def correct_restaurant? restaurant
+    true if @restaurant&.id == restaurant.id
```
Comment (phucmh-1356) [inline app/helpers/application_helper.rb]:
```suggestion
    @restaurant&.id == restaurant.id
```

**Comment by nguyentrison2585** [comment]
Ready

**Pull request merged**